invokeai/app/api/routers/images.py

```diff
@@ -126,3 +147,20 @@ async def list_images(
     """Gets a list of images"""
     result = ApiDependencies.invoker.services.images.list(image_type, page, per_page)
     return result
+
+
+@images_router.delete(
+    "/",
+    operation_id="delete_images",
+)
+async def delete_images(
+    images: list[NonNullableImageField] = Body(
```

Comment:
HTTP `DELETE` generally doesn't guarantee bodies. You shouldn't include a body in a delete, even if it "works on my machine" or "works from me to the server", because some clients or intermediates may not accept/pass it, and it'll be incredibly difficult to debug.

Reply:
(it's also nice in logging to be able to see individual delete requests).

Reply:
Ok, removing this route, and clients will hit the individual image delete routes instead.